src/redux/books/books.js

```diff
@@ -4,21 +4,39 @@ const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
 const firstState = [
   {
     id: 1,
-    title: '',
-    author: '',
-    category: '',
+    title: 'The Hunger Games',
+    author: 'Suzanne Collins',
+    category: 'Action',
   },
   {
     id: 2,
-    title: '',
-    author: '',
-    category: '',
+    title: 'Dune',
+    author: 'Frank Herbert',
+    category: 'Sci-Fi',
   },
   {
     id: 3,
-    title: '',
-    author: '',
-    category: '',
+    title: 'Capital in the Twenty-First Century',
+    author: 'Suzanne Collins',
+    category: 'Economy',
+  },
+  {
+    id: 4,
+    title: 'The Lord of the Rings',
+    author: 'J. R. R. Tolkien',
+    category: 'Fantasy',
+  },
+  {
+    id: 5,
+    title: 'The Hobbit',
+    author: 'J. R. R. Tolkien',
+    category: 'Fantasy',
+  },
+  {
+    id: 6,
+    title: 'A Brief History of Time',
+    author: 'Stephen Hawking',
+    category: 'Sci-Fi',
   },
 ];
```

Comment:
- [ ] Also here you have to make the ids string as they should be like the books that you will add.

```javascript
const firstState = [
  {
    id: '1',
    title: 'The Hunger Games',
    author: 'Suzanne Collins',
    category: 'Action',
  },
  {
    id: '2',
    title: 'Dune',
    author: 'Frank Herbert',
    category: 'Sci-Fi',
  },
  {
    id: '3',
    title: 'Capital in the Twenty-First Century',
    author: 'Suzanne Collins',
    category: 'Economy',
  },
  {
    id: '4',
    title: 'The Lord of the Rings',
    author: 'J. R. R. Tolkien',
    category: 'Fantasy',
  },
  {
    id: '5',
    title: 'The Hobbit',
    author: 'J. R. R. Tolkien',
    category: 'Fantasy',
  },
  {
    id: '6',
    title: 'A Brief History of Time',
    author: 'Stephen Hawking',
    category: 'Sci-Fi',
  },
];
```